includes/admin/class-edd-notices.php

```diff
@@ -355,6 +355,29 @@ private function add_settings_notices() {
 					'is_dismissible' => false
 				) );
 			}
+
+			// Default tax rate detected
+			if ( edd_get_option( 'tax_rate' ) && edd_is_test_mode() ) {
```

Comment:
Are we only showing this notice if we're in `test_mode`? It's likely that our customers will not be in test mode so would never see this notice, if I'm reading this correctly?

Reply:
That was an oops left over from a copy-paste.